### PR TITLE
Add documentation for configuration

### PR DESCRIPTION
**Pull request by azz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -236,18 +232,43 @@ expands the globs rather than your shell, for cross-platform usage.
...
 ```
 
+#### `--resolve-config` and `--config`
+
+If you are repeatedly formatting individual files with `prettier`, you will incur a small performance cost
+when prettier attempts to look up a [configuration file](#configuration-file). In order to skip this, you may
+use:
+
+```bash
+prettier --resolve-config ./my/file.js
```
Comment (vjeux) [inline README.md]: I don't understand what `--resolve-config` does. Does it tell prettier to not resolve the config and trust that the command line arguments are correct?

File: README.md [hunk 2]
```diff
@@ -236,18 +232,43 @@ expands the globs rather than your shell, for cross-platform usage.
...
+
+This will provide you with a path to the configuration file, which you can pass to `--config`:
+
+```bash
+prettier --config ./my/.prettierrc --write ./my/file.js
+```
+
+You can also use `--config` if your configuration file lives somewhere where prettier cannot find it,
+such as a `config/` directory.
+
```
Comment (vjeux) [inline README.md]: Is there a way to say

```js
--no-config
```

to prevent prettier from even looking for it?

File: README.md (hunk 1 above)
Comment (azz) [inline README.md]: No it just outputs the path to the configuration file.

```
$ prettier --resolve-config ./my/file.js
./my/.prettierrc
```

File: README.md (hunk 2 above)
Comment (azz) [inline README.md]: There is now 😉

File: README.md (hunk 1 above)
Comment (CiGit) [inline README.md]: I find it confusing with `resolveConfig` which returns a config object.

Comment (azz) [inline README.md]: Maybe rename to `--find-config`?

Comment (CiGit) [inline README.md]: To me find-config-file would be self explanatory. I find *resolve* a little too broad. You could even rename `resolveConfig` to `getConfig` IMHO

**Pull request merged**